components/date-time/date.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { default as ReactDatePicker } from 'react-datepicker';
+import moment from 'moment';
+
+/**
+ * WordPress dependencies
+ */
+import { settings } from '@wordpress/date';
```

Comment:
I'd prefer if we avoid using the `date` module in the `components` module to keep it as generic as possible?